components/layout/table/construct.rs

```diff
@@ -1026,8 +1026,8 @@ where
                     // script).
                     let (rowspan, colspan) = if info.pseudo_element_type.is_none() {
                         let node = info.node.to_threadsafe();
-                        let rowspan = node.get_rowspan().unwrap_or(1).min(65534) as usize;
```

Comment:
Sorry, I missed this at first: The range is wrong, rowspan can be 0.

https://html.spec.whatwg.org/multipage/tables.html#dom-tdth-rowspan